andela/persephone-ah-frontend
# #166816062 User can reset password

#### What does this PR do?
Add Password Reset

#### Description of Task to be completed?
> - add forgot password page for users to request for a password reset link
> - add password reset page for users to request for password change
> - action and reducers to process password reset request

#### How should this be manually tested?
> - Clone this branch
> - Then run `npm install` and `npm test'
> - Then run `npm run dev' 
> - Navigate to the login page
> - Locate the password reset link and click on it to navigate to the forgot password page
> - Input an existing email then check the mail for a link to reset password

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#166816062](https://www.pivotaltracker.com/story/show/166816062)

#### Screenshots (if appropriate)
NA
#### Questions:
NA
